recipes/libcxxwrap-julia/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libcxxwrap-julia" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "62cb4216efb3dc39736630ec747fb35e8685e6cd888b65d6e431905ba45918c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaInterop/libcxxwrap-julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # Julia has not been packaged for conda on OS X or Windows
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - julia >=0.6,<0.7
+  run:
+    - julia >=0.6,<0.7
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/jlcxx  # [unix]
+    - test -f ${PREFIX}/include/jlcxx/jlcxx.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/JlCxx/JlCxxConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/JlCxx/JlCxxConfigVersion.cmake  # [unix]
+
+    - if exist %LIBRARY_PREFIX%\include\JlCxx\jlcxx.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\JlCxx\JlCxxConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\JlCxx\JlCxxConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/JuliaInterop/CxxWrap.jl
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE.md
+  summary: 'Seamless operability between C++11 and Julia'
+  description: 'Lightweight library that exposes C++ types in Julia and vice versa'
+  doc_url: https://github.com/JuliaInterop/libcxxwrap-julia
+  dev_url: https://github.com/JuliaInterop/libcxxwrap-julia
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - JohanMabille
```

Comment:
Are you ok with being a maintainer, @JohanMabille?

Reply:
Yes